UnitTests/Extensions/Test_BitmapIconExtensionMarkupExtension.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.AppContainer;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace UnitTests.Extensions
+{
+    [TestClass]
+    public class Test_BitmapIconExtensionMarkupExtension
+    {
+        [TestCategory("BitmapIconExtensionMarkupExtension")]
+        [UITestMethod]
+        public void Test_BitmapIconExtension_MarkupExtension_ProvideImage()
+        {
+            var treeroot = XamlReader.Load(@"<Page
+    xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+    xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml""
+    xmlns:ex=""using:Microsoft.Toolkit.Uwp.UI.Extensions"">
+        <Button x:Name=""RootButton"">
+            <Button.Flyout>
+                <MenuFlyout>
+                    <MenuFlyoutItem Icon=""{ex:BitmapIcon Source=/Assets/StoreLogo.png}"" />
```

Comment:
Isn't this supposed to be `ex:BitmapIconExtension`??

Reply:
Marking this as resolved as per our exchange on Twitch 😄

For future reference: https://docs.microsoft.com/en-us/dotnet/desktop-wpf/xaml-services/markup-extensions-overview.